scraper/github/app/scrape_users.py

```diff
@@ -1,7 +1,9 @@
-from app.utils.github_requests import get_users, get_user, request_failed
+from app.utils.github_requests import GithubClient
 from datetime import datetime
 from app.utils.storage import store_user
 
+github_cli = GithubClient()
```

Comment:
Yeah, and i think, that will prevent a memory gap every time we call on the scraping module !

Reply:
as i said before i did the max to not break any function's signature.
and to sanix how a local variable takes less memory than a global variable? in general a Client (a class that consume some resources) is the perfect candidate to be make a global variable because it is normally a const variable. you instantiate it once and just call his methods to get the resources.

https://github.com/jefcolbi/osscameroon-website/blob/main/scraper/github/app/scrape_users.py#L18
if we look this function and make the client local, then we would instanciate it X times we call the function. in this case the client is simple it won't cost anything. but if they were extra initialization steps involved then it would cost us more time.

but if global variables are forbidden then i agree we should instantiate the client locally.

Reply:
Alright there are couple of issues with the fact we are using globals, but the one I want to focus on here is that if our main function does a get_users and a get_projects we would create 2 clients as both module creates their own `clients`.I was not happy seing the introduction of an object here anyway as it was for me unecessary, but since you created one, I think we should then use properly and I don't mind changing the function signature if that's what your implementation require